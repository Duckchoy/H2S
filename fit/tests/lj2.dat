0 0.000175813389875      -0.240403998522525      0.000060318030875      -0.227977760441375      -19473.7440847660      0.183454670180187
0 0.000092182491025      -0.183442153938825      0.000034024986075      -0.181217285312200      -19473.5097301091      0.181134859003763
0 0.000049860284075      -0.142181075095025      0.000019791865975      -0.145844729991500      -19473.0626142083      0.157694766608642
0 0.000015916676050      -0.089042888161775      0.000007256638850      -0.097566945224425      -19472.3065436056      0.157256413877736
0 0.000009362139950      -0.071771090191525      0.000004551507000      -0.080936640643075      -19471.4234401913      0.127487392760343
0 0.000001752986425      -0.036506867970250      0.000001035711450      -0.044722439537450      -19472.2609793814      0.161468108687617
0 0.000000613112900      -0.023939936568850      0.000000405524625      -0.030707137126825      -19471.3614887622      0.171103862579763
0 0.000000099160150      -0.011523439716550      0.000000078128450      -0.015850882642400      -19471.8054762213      0.143659329830843

