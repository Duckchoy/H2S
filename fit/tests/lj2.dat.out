prob -22.15205028

 new best -19471.81028 512.3404205 50.40587356 971.3200205 -42.66950969 -22.15205028
prob -25.93883583
prob -22.29752182
prob -31.61523757
prob -22.25543897
prob -464.0324762
prob -22.55549002
prob -36.78595182
prob -23.68791443
prob -22.24368137
prob -30.30679834
prob -25.92327239
prob -24.75307846
prob -22.4575798
prob -22.19780128
prob -31.15932117
prob -29.06148869
prob -22.43878663
prob -22.48202814
prob -26.37180542
prob -23.59544438
prob -40.79719776
prob -22.31527436
prob -23.87254496
prob -22.13540121

 new best -19471.7561 817.8792757 45.47349595 -406.6294324 -37.46560909 -22.13540121
prob -23.19929202
prob -22.83665345
prob -27.98571238
prob -35.9219904
prob -26.40247865
prob -22.59664755
prob -29.36341659
prob -22.31538702
prob -32.89468088
prob -22.65860993
prob -24.74677299
prob -27.98256751
prob -33.42462985
prob -23.5914207
prob -22.65695675
prob -27.56177737
prob -23.16280801
prob -25.92531336
prob -22.16992182
prob -23.8093195
prob -26.50075269
prob -22.61548728
prob -22.75653078
prob -22.53457536
prob -29.21938384
prob -22.52606194
prob -335.5263932
prob -22.53955528
prob -22.59083525
prob -30.99176977
prob -23.75144357
prob -25.5288697
prob -22.46000137
prob -22.60878057
prob -23.25955217
prob -41.8082906
prob -24.02978782
prob -26.71077014
prob -22.33790732
prob -70.03898328
prob -22.80187996
prob -22.46979231
prob -22.16299989
prob -28.18175111
prob -22.19179489
prob -26.88039638
prob -25.74891164
prob -22.52671185
prob -31.91848636
prob -24.06740058
prob -22.46872012
prob -22.30638766
prob -28.18899875
prob -22.8089671
prob -22.66756647
prob -22.37686896
prob -23.23207196
prob -23.52229617
prob -36.81733466
prob -26.10791147
prob -22.65822157
prob -22.74970888
prob -27.08631419
prob -22.38613481
prob -22.56704338
prob -22.45993368
prob -22.6149761
prob -22.98575573
prob -22.44847113
prob -22.55835776
prob -27.65025321
prob -25.56561641
prob -24.00810171
prob -23.03012911
prob -37.48851562
prob -32.53694454
prob -36.81538525
prob -22.15157228
prob -23.9794818
prob -22.88534778
prob -24.4140278
prob -22.43922685
prob -28.69523357
prob -32.69524856
prob -22.67126094
prob -22.65064179
prob -22.46362369
prob -22.48520211
prob -26.96101607
prob -27.9681077
prob -25.53159262
prob -73.16683018
prob -23.08954344
prob -23.39734031
prob -21.9099201

 new best -19471.80545 1553.953985 52.82160888 -197.7761068 -44.70312427 -21.9099201
prob -22.017182
prob -22.3018903
prob -22.5755338
prob -28.58570699
prob -26.21866892
prob -22.63603094
prob -22.53514743
prob -22.40720238
prob -22.59674632
prob -22.30958021
prob -25.14986745
prob -22.98990321
prob -31.49763494
prob -21.97640047
prob -22.33902381
prob -23.30645163
prob -98.55079859
prob -22.7795379
prob -22.48327565
prob -22.34802259
prob -143.6565411
prob -27.51713437
prob -23.35694484
prob -22.4054128
prob -22.7026524
prob -22.78350224
prob -22.12627007
prob -22.48928487
prob -22.3541892
prob -23.02446401
prob -36.25263939
prob -22.65587136
prob -29.87960453
prob -22.35014182
prob -24.16306281
prob -24.07170269
prob -22.41177373
prob -30.65111418
prob -22.54259275
prob -22.19069233
prob -27.82084222
prob -22.37883626
prob -40.96472643
prob -27.19946873
prob -22.86834923
prob -23.16482266
prob -23.43821043
prob -22.76739086
prob -22.03303607
prob -23.05278882
prob -22.61679259
prob -24.12085015
prob -22.20482246
prob -23.09955839
prob -34.87804614
prob -22.16855553
prob -22.28251711
prob -22.38649086
prob -22.32463589
prob -22.31586825
prob -22.54630811
prob -22.53877461
prob -30.98122648
prob -24.6765244
prob -22.56169172
prob -22.61704494
prob -24.58738178
prob -23.00791541
prob -22.37152959
prob -23.51573481
prob -22.55151662
prob -23.55177468
prob -29.37892162
prob -41.81120918
prob -22.28926759
prob -22.04945381
prob -22.4836179
prob -23.61026698
prob -22.92753215
prob -24.25709799
prob -22.60638293
prob -22.39227044
prob -23.65210421
prob -22.30879895
prob -37.74040073
prob -22.16437188
prob -23.80386486
prob -33.71069515
prob -22.37253022
prob -22.50210451
prob -23.15488273
prob -22.34344275
prob -23.24851193
prob -23.91030261
prob -22.34009905
prob -22.52024113
prob -22.5154856
prob -22.788193
prob -22.15411982
prob -23.70304885
prob -22.54842441
prob -22.69636857
prob -22.2767702
prob -23.36706386
prob -22.22131242
prob -27.67520232
prob -22.22954279
prob -22.3075527
prob -33.24900955
prob -32.02675205
prob -60.42450114
prob -22.46957775
prob -22.67071067
prob -35.15542473
prob -29.4399048
prob -22.98687828
prob -22.36834852
prob -22.58680533
prob -22.02448802
prob -22.56903718
prob -22.94765214
prob -22.63504001
prob -22.6314248
prob -22.62008489
prob -22.49361903
prob -23.37204653
prob -22.20007373
prob -22.6136013
prob -22.86355638
prob -25.21418898
prob -25.26012437
prob -22.53586883
prob -31.51748666
prob -46.35479137
prob -22.16232254
prob -30.8969107
prob -23.72568631
prob -22.0129715
prob -22.46922753
prob -31.26112447
prob -22.39104664
prob -31.71672389
prob -31.67587944
prob -23.00632576
prob -22.59000185
prob -22.2874498
prob -23.27110457
prob -22.63092955
prob -22.43986341
prob -32.11817522
prob -23.6241995
prob -36.17402135
prob -33.74022241
prob -22.67495228
prob -26.5397279
prob -22.35815004
prob -24.01422685
prob -22.35003515
prob -22.43305413
prob -22.5349891
prob -22.44212971
prob -22.97957952
prob -22.82516192
prob -22.49381334
prob -22.44344406
prob -22.44397637
prob -23.07085167
prob -23.268525
prob -23.68294334
prob -22.59478912
prob -22.67207291
prob -36.08516664
prob -22.30951421
prob -23.9293674
prob -23.24728839
prob -36.27932967
prob -22.29977629
prob -22.17061364
prob -22.48106402
prob -22.69838986
prob -29.34466311
prob -26.96415877
prob -39.50360207
prob -33.46450267
prob -25.09075529
prob -35.42174912
prob -23.71193551
prob -22.47290177
prob -26.41896325
prob -69.18414027
prob -22.36980768
prob -22.29198601
prob -22.33129627
prob -24.01551761
prob -23.81853128
prob -22.44528971
prob -32.68717547
prob -41.8036552
prob -22.38106565
prob -32.38834099
prob -23.297621
prob -23.93592382
prob -22.26977866
prob -22.63110838
prob -22.63426692
prob -22.06684063
prob -22.26674175
prob -22.534964
prob -22.31781944
prob -23.40224091
prob -30.63697766
prob -26.01976738
prob -23.05107033
prob -22.46085192
prob -25.67475284
prob -22.54606897
prob -23.22865602
prob -22.2343613
prob -34.63263359
prob -22.48167869
prob -34.55447671
prob -33.92183842
prob -44.38995437
prob -22.51931983
prob -22.29395963
prob -27.68852957
prob -23.41030339
prob -30.85769656
prob -26.70085991
prob -32.16611629
prob -22.60858102
prob -22.3855614
prob -65.70643126
prob -46.71816639
prob -26.68962872
prob -22.45625161
prob -22.50394377
prob -22.15475216
prob -22.42201178
prob -22.67917231
prob -22.73664229
prob -22.48386553
prob -22.40454701
prob -22.90656789
prob -22.58076947
prob -35.49718798
prob -23.97102152
prob -26.47988904
prob -32.21961422
prob -22.3260278
prob -27.69032551
prob -22.33404928
prob -22.32169929
prob -22.12715779
prob -23.37828223
prob -25.75738285
prob -22.53289645
prob -22.1731925
prob -30.49913278
prob -30.61111787
prob -22.50347357
prob -29.97963164
prob -22.39624034
prob -25.29388981
prob -23.73284304
prob -22.78328533
prob -22.96154159
prob -34.19178491
prob -34.39229564
prob -22.46395887
prob -44.34591985
prob -22.58172989
prob -33.95501243
prob -26.29517753
prob -22.33369749
prob -22.38261883
prob -24.42765072
prob -25.16175692
prob -22.50371892
prob -22.75394634
prob -23.00497166
prob -22.54550464
prob -32.05497877
prob -23.00967332
prob -22.74473916
prob -23.01026317
prob -22.39957597
prob -30.75076956
prob -22.4541206
prob -27.31080091
prob -22.43015261
prob -22.40654947
prob -22.53161018
prob -22.44094359
prob -25.98895572
prob -24.04339167
prob -22.30510755
prob -22.6477972
prob -22.34768245
prob -22.41411018
prob -29.67086483
prob -22.41859488
prob -30.09002415
prob -24.65327216
prob -32.34494822
prob -22.1805588
prob -25.0682789
prob -27.34930382
prob -22.60164511
prob -27.76244954
prob -34.54096043
prob -25.24408505
prob -27.9296729
prob -22.35104921
prob -22.32788279
prob -22.10680223
prob -23.2090693
prob -28.02648669
prob -23.03254897
prob -30.17009441
prob -22.36259803
prob -23.99688383
prob -22.20842345
prob -28.66723564
prob -22.34766366
prob -22.44959566
prob -22.58960332
prob -22.39633683
prob -22.60989995
prob -22.31954183
prob -22.24530309
prob -27.03169987
prob -22.65644346
prob -30.32016835
prob -23.02514139
prob -143.409429
prob -30.9660088
prob -23.76647105
prob -22.61579707
prob -22.50443541
prob -22.91606726
prob -22.53274686
prob -22.30844118
prob -22.53779986
prob -26.24533904
prob -61.44183596
prob -24.68872427
prob -22.60126583
prob -22.3416124
prob -22.51525809
prob -22.34954534
prob -22.45553155
prob -27.44545629
prob -22.63618415
prob -22.80143825
prob -22.58584887
prob -22.77485463
prob -22.33174935
prob -22.45720441
prob -23.07386696
prob -23.02540072
prob -38.34708547
prob -22.86079892
prob -23.21160358
prob -22.32787637
prob -22.19067067
prob -22.52787565
prob -38.47795696
prob -22.53857092
prob -22.50841969
prob -22.51273293
prob -22.21002254
prob -25.98398934
prob -22.53636791
prob -32.545911
prob -22.57785467
prob -25.43276369
prob -22.4912181
prob -22.61261462
prob -31.81674006
prob -23.0858516
prob -22.94782095
prob -22.51513717
prob -22.52092889
prob -32.83734649
prob -22.67172488
prob -25.25426896
prob -34.87360808
prob -22.38055719
prob -24.75630734
prob -38.66736408
prob -22.22635624
prob -33.90573398
prob -24.97977793
prob -22.25127498
prob -22.18197085
prob -29.93227458
prob -22.74987646
prob -22.71407556
prob -22.22151515
prob -22.28073751
prob -21.96834996
prob -22.35735371
prob -22.53929585
prob -25.88634443
prob -22.70608769
prob -22.11184326
prob -36.04838983
prob -24.29688224
prob -27.29855355
prob -28.72581188
prob -23.37947458
prob -23.81998928
prob -27.70945117
prob -22.4807353
prob -22.64556721
prob -22.20342885
prob -62.10081318
prob -26.46035078
prob -22.68775839
prob -38.36599788
prob -30.39171232
prob -32.01968232
prob -22.51885824
prob -22.36473445
prob -33.90613866
prob -22.60688723
prob -29.99841776
prob -22.79950539
prob -23.86641244
prob -41.85906543
prob -22.51661852
prob -22.35723529
prob -30.47844531
prob -27.49489853
prob -33.19396057
prob -22.1820505
prob -22.60636937
prob -22.30520172
prob -23.72846888
prob -24.05733091
prob -29.72608176
prob -22.2533179
prob -25.30420853
prob -22.42944057
prob -28.88920886
prob -22.47146344
prob -22.20294586
prob -22.3139687
prob -24.25508014
prob -42.56626367
prob -27.75661047
prob -32.32814834
prob -22.50821947
prob -22.60000032
prob -25.16641227
prob -22.20654615
prob -22.34104962
prob -24.53104922
prob -22.60421122
prob -22.49286408
prob -49.11725452
prob -22.6953419
prob -24.92438651
prob -182.7757023
prob -22.49742095
prob -26.00535659
prob -26.06120992
prob -23.23000358
prob -22.40094432
prob -22.72282157
prob -22.40200119
prob -22.50269274
prob -28.26698115
prob -23.39769238
prob -22.83285327
prob -22.31999735
prob -22.53797036
prob -22.19053469
prob -26.42323823
prob -23.79249174
prob -30.03701188
prob -22.24667183
prob -24.26459665
prob -70.51395046
prob -38.4391522
prob -23.52631357
prob -22.43338527
prob -22.5795749
prob -22.08497365
prob -25.21338322
prob -24.68468627
prob -37.15107223
prob -22.36343769
prob -29.8586618
prob -23.30227786
prob -34.22839649
prob -22.23305751
prob -22.49993942
prob -33.71376527
prob -30.98499583
prob -24.40663712
prob -22.30243457
prob -28.840295
prob -22.43458843
prob -29.63125681
prob -23.20535668
prob -22.57430971
prob -22.39765246
prob -22.39519547
prob -22.31329021
prob -37.48953399
prob -24.79121778
prob -32.74731449
prob -22.29193791
prob -22.27957964
prob -22.88683397
prob -24.78055521
prob -23.29322229
prob -22.28790652
prob -48.02902976
prob -22.27536471
prob -22.50869558
prob -23.22912539
prob -23.9653987
prob -22.34459662
prob -22.15866825
prob -31.49470215
prob -37.51559596
prob -29.8229411
prob -28.18764084
prob -30.25031146
prob -23.10125139
prob -24.30549548
prob -22.49040329
prob -28.13806929
prob -35.14681589
prob -22.12348623
prob -32.46454192
prob -31.02148682
prob -24.62768902
prob -23.91418059
prob -22.42411849
prob -24.57895534
prob -22.41953662
prob -31.74836415
prob -22.57719183
prob -22.32999231
prob -34.10773008
prob -22.60109215
prob -22.29011416
prob -26.81128458
prob -23.00256997
prob -22.28343461
prob -22.61701706
prob -29.21047419
prob -22.33701044
prob -22.33575882
prob -22.65110677
prob -24.02959581
prob -22.49974022
prob -45.40782618
prob -31.17973231
prob -30.09462457
prob -24.27297517
prob -22.61589626
prob -22.51021806
prob -40.94387775
prob -22.47466744
prob -22.80340181
prob -28.32041336
prob -25.15944277
prob -22.27615138
prob -23.00878886
prob -24.08703133
prob -26.27362023
prob -22.56097658
prob -29.09072913
prob -22.23518434
prob -22.43400666
prob -30.19185081
prob -33.09507543
prob -27.63815689
prob -22.52422488
prob -22.1390219
prob -22.54385559
prob -22.12792187
prob -22.09049256
prob -22.73745062
prob -22.84210773
prob -22.52660631
prob -24.47212167
prob -22.07424291
prob -22.71019489
prob -22.47759195
prob -32.68819359
prob -24.86736899
prob -22.76474251
prob -32.66367306
prob -26.79228301
prob -22.15720916
prob -23.03884847
prob -104.5615875
prob -22.39807937
prob -22.66992464
prob -22.77145998
prob -24.1331595
prob -22.42060795
prob -22.49013175
prob -22.22415912
prob -22.58543463
prob -22.50509937
prob -22.22833089
prob -23.49508
prob -25.13420018
prob -22.67114471
prob -22.18543009
prob -22.44688571
prob -23.13791996
prob -22.29005209
prob -27.14755156
prob -34.29613377
prob -22.4294965
prob -36.71388455
prob -22.67301141
prob -23.29553601
prob -25.94068087
prob -22.34394124
prob -22.35413066
prob -34.40718987
prob -22.23864689
prob -22.52330775
prob -23.93455493
prob -22.83590223
prob -22.36054935
prob -22.40097264
prob -22.41682242
prob -29.15763836
prob -30.51416039
prob -22.44237658
prob -23.56389809
prob -26.67285879
prob -22.67974908
prob -22.29289121
prob -22.47813316
prob -53.95660666
prob -22.56301034
prob -33.82834063
prob -22.57913339
prob -23.96211295
prob -23.72483603
prob -36.56105553
prob -22.63343549
prob -25.83597362
prob -25.19251854
prob -39.06388275
prob -22.13445586
prob -25.62616691
prob -22.47688781
prob -24.61275914
prob -22.35170903
prob -22.16606171
prob -22.37665597
prob -23.10984649
prob -22.47422966
prob -22.28187229
prob -22.46608044
prob -29.75095551
prob -22.37679519
prob -25.26987221
prob -24.68345614
prob -22.82623459
prob -22.41529364
prob -23.26362762
prob -22.49421464
prob -29.16501664
prob -32.92671693
prob -24.38719143
prob -25.69339889
prob -22.25880803
prob -22.79775051
prob -22.35852987
prob -22.3342368
prob -24.61134114
prob -23.67791046
prob -22.54467006
prob -27.37490166
prob -22.41268181
prob -22.32195368
prob -22.93828074
prob -22.46310906
prob -31.49908059
prob -23.30367415
prob -22.0676067
prob -42.15689419
prob -22.55191082
prob -36.49053643
prob -22.72777454
prob -31.60295173
prob -22.11461595
prob -23.10294517
prob -35.39300301
prob -35.1639322
prob -22.58023449
prob -61.67315783
prob -29.26512561
prob -22.23447323
prob -22.89039172
prob -26.49340363
prob -29.83982771
prob -23.98955772
prob -25.20631811
prob -22.47448691
prob -25.27293476
prob -22.34996077
prob -22.46772183
prob -25.31410524
prob -25.14998512
prob -28.48956856
prob -22.3468618
prob -23.61261266
prob -24.39815331
prob -22.45973093
prob -23.39928609
prob -22.30418996
prob -25.74538266
prob -23.24865245
prob -22.41603072
prob -25.17743012
prob -187.8074155
prob -22.52164854
prob -22.3231426
prob -22.27132143
prob -38.73375146
prob -34.53048786
prob -22.45636289
prob -27.92597243
prob -22.19132304
prob -22.40097689
prob -22.27471146
prob -22.26768925
prob -42.33044859
prob -22.51228075
prob -24.52469236
prob -22.31265811
prob -22.47277057
prob -22.32373871
prob -22.45293762
prob -34.32477403
prob -33.68921812
prob -26.24586516
prob -22.55276984
prob -23.68893824
prob -30.20236699
prob -23.91052407
prob -27.99637304
prob -40.12819509
prob -22.67711918
prob -22.41685698
prob -24.63174739
prob -23.93147512
prob -22.72522175
prob -22.27581125
prob -37.9909036
prob -46.24595315
prob -22.40992407
prob -24.61479003
prob -23.85768207
prob -31.27268235
prob -22.5055596
prob -25.53788342
prob -104.4850512
prob -23.0052563
prob -22.6033443
prob -22.48069213
prob -22.97376295
prob -26.87377292
prob -25.11348684
prob -22.4445571
prob -22.83228015
prob -25.50649209
prob -33.02413579
prob -23.26165871
prob -22.70580965
prob -22.44785104
prob -22.35611669
prob -23.60430155
prob -22.05216155
prob -25.11552798
prob -22.29327538
prob -28.8644313
prob -23.76565531
prob -22.78495571
prob -34.21370349
prob -23.56476913
prob -22.5578144
prob -28.32944864
prob -23.21144743
prob -22.55725455
prob -22.83059045
prob -26.3581711
prob -30.67091865
prob -22.4430288
prob -23.39826812
prob -22.36404878
prob -22.48580751
prob -22.01616196
prob -22.2484233
prob -58.8961653
prob -22.28584845
prob -37.45298886
prob -36.36923697
prob -25.40614557
prob -22.55751934
prob -21.95098504
prob -22.35651317
prob -22.4515318
prob -23.09580012
prob -22.12162817
prob -22.34093254
prob -24.39781715
prob -22.4484713
prob -22.44852741
prob -27.49345119
prob -22.14370723
prob -22.46463062
prob -22.60592609
prob -22.37618346
prob -27.19067605
prob -22.64965909
prob -22.31620782
prob -24.43238402
prob -23.36187047
prob -23.63068058
prob -23.0346867
prob -22.49453553
prob -23.0275005
prob -39.24805524
prob -23.64309224
prob -23.2632185
prob -22.45475599
prob -24.77445488
prob -118.8779037
prob -22.31977509
prob -22.34865777
prob -22.36480083
prob -24.01642896
prob -22.46136087
prob -22.4489965
prob -29.90300989
prob -23.48293396
prob -23.91537203
prob -28.49749081
prob -22.50520996
prob -22.45311707
prob -28.07624591
prob -22.4236336
prob -22.43535218
prob -23.08759972
prob -22.44698747
prob -22.10850386
prob -23.219142
prob -22.19374175
prob -22.17188811
prob -24.60408812
prob -22.31233171
prob -22.49393486
prob -26.87124104
prob -22.83799639
prob -27.59941433
prob -22.43087986
prob -24.02515082
prob -22.13231935
prob -30.52848028
prob -22.32239703
prob -22.54429249
prob -23.03899085
prob -22.24956167
prob -27.2023086
prob -26.88932789
numerical derivative 0 analytic 0
numerical derivative 0 analytic 0
numerical derivative 0 analytic 0
numerical derivative 0 analytic 0
numerical derivative 0 analytic 0
acceptance 0.715862
timesteps   avg   err
0.007435654507  -19471.81349  0.1331354149
1.039866213  1394.579025  85.82838789
0.2030946244  52.90787044  8.285901057
1.024823694  -81.25026709  47.06183435
0.3285680712  -44.91072081  8.905070221

c[0] = -19471.81349 +/- 0.1331354149
c[1] = 1394.579025 +/- 85.82838789
c[2] = 52.90787044 +/- 8.285901057
c[3] = -81.25026709 +/- 47.06183435
c[4] = -44.91072081 +/- 8.905070221
Assessment of the fit:
fitted value -19474.05382 data -19473.74408 +/- 0.1834546702
fitted value -19473.25463 data -19473.50973 +/- 0.181134859
fitted value -19472.71807 data -19473.06261 +/- 0.1576947666
fitted value -19472.12115 data -19472.30654 +/- 0.1572564139
fitted value -19471.96313 data -19471.42344 +/- 0.1274873928
fitted value -19471.73411 data -19472.26098 +/- 0.1614681087
fitted value -19471.7002 data -19471.36149 +/- 0.1711038626
fitted value -19471.71116 data -19471.80548 +/- 0.1436593298
minimum skew nan
